src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+
+    @Override
+    public void handleOperation(FruitTransaction fruitTransaction, Storage storage) {
+        storage.getInventory().merge(
+                fruitTransaction.getFruit(),
+                fruitTransaction.getQuantity(),
+                Integer::sum
+        );
+    }
```

Comment:
The `BalanceOperation` should set the balance of a fruit to a particular value, not sum it up. According to the checklist, the balance operation should replace the current quantity with the new quantity provided in the transaction.